src/configurationProvider.ts

```diff
@@ -386,8 +377,8 @@ interface IMainClassOption {
 }
 
 interface IValidationResult {
-    readonly passed: boolean;
-    readonly message?: string;
+    readonly isValid: boolean;
+    readonly errorMessage?: string;
```

Comment:
The message field can be neutral because there could message for both valid and invalid case.

Reply:
Agree. Have reverted the field back to `message`.